GameBling/GameBling/ChooseAvatar/ChooseAvatarViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ChooseAvatarViewModel.swift
+//  GameBling
+//
+//  Created by Dong Duong V. on 11/09/2023.
+//
+
+import Foundation
+import UIKit
+
+final class ChooseAvatarViewModel {
+
+    var selectedUserInfo: UserInfor?
+    var dataUsers: [UserInfor] =
+    [
+        UserInfor(id: 1, image: UIImage(named: "image1"), gender: Gender.male),
+        UserInfor(id: 2, image: UIImage(named: "image2"), gender: Gender.felmale),
+        UserInfor(id: 3, image: UIImage(named: "image3"), gender: Gender.male),
+        UserInfor(id: 4, image: UIImage(named: "image4"), gender: Gender.felmale),
+        UserInfor(id: 5, image: UIImage(named: "image5"), gender: Gender.male),
+        UserInfor(id: 6, image: UIImage(named: "image6"), gender:  Gender.male)
+    ]
```

Comment:
Please move it to "Model". (Make another class in Model group to hold the data)
Follow MVVM structure.